src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperationTest.java

```diff
@@ -610,6 +611,264 @@ public void findWithDynamic() throws Exception {
       assertThat(result.errors()).isNullOrEmpty();
     }
 
+    @Test
```

Comment:
Would perhaps make sense to create a new `@Nested` grouping for these? (maybe including existing `findWithDynamic`)